src/recipes/FactoryBuildings.json

```diff
@@ -272,10 +356,235 @@
       "requiresSupplementalResource": false
     }
   },
+  {
+    "id": "Build_PipelineSupport_C",
+    "name": "Pipeline Support",
+    "index": 12,
+    "description": "Connects Pipeline segments. Support height can be adjusted.\r\nUseful for routing Pipelines more precisely and across long distances.",
+    "minimumPowerConsumption": null,
+    "maximumPowerConsumption": null,
+    "averagePowerConsumption": null,
+    "powerConsumption": null,
+    "powerConsumptionExponent": null,
+    "somersloopPowerConsumptionExponent": null,
+    "somersloopSlots": null,
+    "clearance": {
+      "width": null,
+      "length": null,
+      "height": null
+    },
+    "imagePath": "/images/game/null",
```

Comment:
We should probably handle this in a better way (items without images), but for now they are just not used. Could you add a `// TODO` on the parser where you added the `if (!resourcePath).. `? Cause it could break something in the future

Reply:
I guess, add some issue, but out of scope for this pr.